### DISCUSSION TITLE
Rename Azure AD to Microsoft Entra ID in docs

### DISCUSSION THREAD

**Discussion by tehho** [community/community#86499] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The docs for "Managing SAML single sign-on for your organization" reference Azure AD which is renamed to Microsoft Entra ID. 
https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/about-identity-and-access-management-with-saml-single-sign-on#supported-saml-services
Please update your documentation

**Comment by chrisreddington** [comment]
Hi there, thanks for flagging this!

I've passed this onto our docs team -- This is on their backlog. Thank you 🎉
